### PR TITLE
[FLINK-1891]Add the input storageDirectory empty check

### PR DESCRIPTION
**Pull request by matadorhong**
Add the input storageDirectory empty check, if input of storageDirectory is empty, we should use tmp as the base dir
And also fix the code style proble of blank space between every ) and {

### CODE REVIEW

File: flink-runtime/src/main/java/org/apache/flink/runtime/blob/BlobUtils.java [hunk 1]
```diff
@@ -64,15 +64,19 @@
 	 * @return the storage directory used by a BLOB service
 	 */
 	static File initStorageDirectory(String storageDirectory) {
-		File baseDir = storageDirectory != null ?
-				new File(storageDirectory) :
-				new File(System.getProperty("java.io.tmpdir"));
+		File baseDir;
+		if (storageDirectory == null || storageDirectory.isEmpty()) {
```
Comment (fhueske) [inline flink-runtime/src/main/java/org/apache/flink/runtime/blob/BlobUtils.java]: can you change `storageDirectory.isEmpty()` to `storageDirectory.trim().isEmpty()` to remove all whitespaces before the empty-check?

**Comment by fhueske** [comment]
Thanks for the PR. 
Looks good to me, except for a possible minor improvement.

File: flink-runtime/src/main/java/org/apache/flink/runtime/blob/BlobUtils.java (hunk 1 above)
Comment (matadorhong) [inline flink-runtime/src/main/java/org/apache/flink/runtime/blob/BlobUtils.java]: Yeah, you are right. Thank for you notes.I have change it.

**Comment by fhueske** [comment]
I'm going to merge this PR.

**Pull request closed without merging**